python/bashrc

```diff
@@ -0,0 +1,19 @@
+VIRTUAL_ENV_PATH=$(pwd)/venv
```

Comment:
This sets a dev env if it doesn't exist on $(pwd)/venv when `okteto up` finishes the session.  Then it 'automagicall' appears and it's activated, so the dev doesn't have to create and activate it.